hera_mc/utils.py

```diff
@@ -27,6 +32,53 @@ def bytes_to_str(b):
         return b.decode('utf8')
 
 
+
+
+def LSTScheduler(starttime, LSTbin_size, longitude=21.25):
+    """
+    Round a time to the nearest LST bin.
+
+    LSTbins start at 0 and step according to LSTbin_size.
+
+    Inputs:
+        starttime: astropy time object
```

Comment:
This isn't quite numpy format. see https://numpydoc.readthedocs.io/en/latest/format.html

`Inputs` should be `Parameters` with a line of hyphens below. The parameter names are not indented relative to `Parameters`. There should be spaces on either side of the colon and only the type goes on the same line as the parameter name. A sentence describing the parameter goes on the next line, indented. The `Returns` section should be followed by a line of hyphens and the first line below that is not indented and either gives only the type or the name of the returned value followed by ` : ` and the type. The description of the returned value is indented on the next line.

 These details are small but important for automated documentation detection.

Reply:
ok, I pushed up some edits to get more numpy

Reply:
much better. The only weirdness now is the indented lines. They should just be indented once relative to the line above, not made to line up vertically in any weird way. Do you want me to fix that?

Reply:
like this?
```
    Parameters
    -----
    starttime : astropy.time.Time
               Target schedule time
    LSTbin_size : float
                 lst bin size in seconds
    longitude : float
               telescope longitude in degrees

    Returns
    -----
    schedule time :  astropy.time.Time
                              time of next LST bin
    schedule sidereal time :  astropy.coord.Angle
                              sidereal time of next LST bin
```

Reply:
No, that's even more indented. It should be like this:

```
    Parameters
    -----
    starttime: astropy.time.Time
        Target schedule time
    LSTbin_size: float
        lst bin size in seconds
    longitude: float
        telescope longitude in degrees

    Returns
    -----
    schedule time :  astropy.time.Time
        time of next LST bin
    schedule sidereal time :  astropy.coord.Angle
        sidereal time of next LST bin
```

Reply:
thanks. doing it myself so I learn.

Reply:
updated with less indenting.

Reply:
did you push it? It's still much more indented than mine. There should be just 4 spaces extra relative to the lines giving the names and types.